eos/statistics/log-likelihood.cc

```diff
@@ -1362,169 +1345,19 @@ namespace eos
-                                             const unsigned & number_of_observations = 6u);
-    template LogLikelihoodBlockPtr LogLikelihoodBlock::MultivariateGaussian<7>(ObservableCache cache, const std::array<ObservablePtr, 7> & observables,
-                                             const std::array<double, 7> & mean, const std::array<std::array<double, 7>, 7> & covariance,
-                                             const unsigned & number_of_observations = 7u);
-    template LogLikelihoodBlockPtr LogLikelihoodBlock::MultivariateGaussian<8>(ObservableCache cache, const std::array<ObservablePtr, 8> & observables,
-                                             const std::array<double, 8> & mean, const std::array<std::array<double, 8>, 8> & covariance,
-                                             const unsigned & number_of_observations = 8u);
-    template LogLikelihoodBlockPtr LogLikelihoodBlock::MultivariateGaussian<9>(ObservableCache cache, const std::array<ObservablePtr, 9> & observables,
-                                             const std::array<double, 9> & mean, const std::array<std::array<double, 9>, 9> & covariance,
-                                             const unsigned & number_of_observations = 9u);
-    template LogLikelihoodBlockPtr LogLikelihoodBlock::MultivariateGaussian<11>(ObservableCache cache, const std::array<ObservablePtr, 11> & observables,
-                                             const std::array<double, 11> & mean, const std::array<std::array<double, 11>, 11> & covariance,
-                                             const unsigned & number_of_observations = 11u);
-    template LogLikelihoodBlockPtr LogLikelihoodBlock::MultivariateGaussian<12>(ObservableCache cache, const std::array<ObservablePtr, 12> & observables,
-                                             const std::array<double, 12> & mean, const std::array<std::array<double, 12>, 12> & covariance,
-                                             const unsigned & number_of_observations = 12u);
-    template LogLikelihoodBlockPtr LogLikelihoodBlock::MultivariateGaussian<16>(ObservableCache cache, const std::array<ObservablePtr, 16> & observables,
-                                             const std::array<double, 16> & mean, const std::array<std::array<double, 16>, 16> & covariance,
-                                             const unsigned & number_of_observations = 16u);
-    template LogLikelihoodBlockPtr LogLikelihoodBlock::MultivariateGaussian<18>(ObservableCache cache, const std::array<ObservablePtr, 18> & observables,
-                                             const std::array<double, 18> & mean, const std::array<std::array<double, 18>, 18> & covariance,
-                                             const unsigned & number_of_observations = 18u);
-    template LogLikelihoodBlockPtr LogLikelihoodBlock::MultivariateGaussian<19>(ObservableCache cache, const std::array<ObservablePtr, 19> & observables,
```

Comment:
I'm glad all these explicit instantiations are gone. They were a pain to add

Reply:
Me too!